transfer-archive/app/helpers/filewriter.py

```diff
@@ -0,0 +1,21 @@
+import logging
+
+from app.settings import Settings
+
+logger = logging.getLogger(__name__)
+
+
+def write_rclone_config(config: dict):
+    settings = Settings()
+    try:
+        with open(settings.rclone_config_path, 'a+') as result_file:
+            for key in config:
+                if key == "name":
+                    result_file.write(f"\n[{config[key]}]\n")
+                else:
+                    result_file.write(f"{key} = {config[key]}\n")
+    except FileNotFoundError as error:
+        logger.error(f"Application failed writing to file with error: {error}")
+        raise
+    except Exception as error:
+        logger.error(error)
```

Comment:
En mulig løsning for å bruke SecretStr, ref diskusjonen https://github.com/arkivverket/mottak/pull/1029/files#r769811236
```suggestion
import logging

from pydantic import SecretStr

from app.settings import Settings

logger = logging.getLogger(__name__)


def write_rclone_config(config: dict):
    settings = Settings()
    try:
        with open(settings.rclone_config_path, 'a+') as result_file:
            for key in config:
                value = config[key]
                if isinstance(value, SecretStr):
                    value = value.get_secret_value()

                if key == "name":
                    result_file.write(f"\n[{value}]\n")
                else:
                    result_file.write(f"{key} = {value}\n")
    except FileNotFoundError as error:
        logger.error(f"Application failed writing to file with error: {error}")
        raise
    except Exception as error:
        logger.error(error)

```

Reply:
Utestet såklart 😸 😈

Reply:
Testet lokalt og sett at `secret_access_key` skrives dekryptert til config filen